src/Event.js

```diff
@@ -107,7 +107,7 @@ class KeyboardEvent extends Event {
     if (init.key === undefined || init.code === undefined) {
       const keySpec = findKeySpecByKeyCode(init.keyCode);
       if (keySpec) {
-        init.key = keySpec.key;
+        init.key = init.shiftKey !== undefined?keySpec.shiftKey:keySpec.key;
```

Comment:
Can we clean up the spacing to be consistent with this file?

Reply:
Yeh for sure, i just noticed this breaks with/without caps lock. i need to investigate more